Tests/FermionMatrix/test_fermionMatrixHubbardExp.cpp

```diff
@@ -404,7 +406,7 @@ void test_logDetM_phi_plus_two_pi(const NSL::size_t nt, LatticeType & Lattice, c
     NSL::FermionMatrix::HubbardExp M(Lattice,nt,beta);
     M.populate(phi);
     NSL::FermionMatrix::HubbardExp Mshift(Lattice,nt,beta);
-    M.populate(phiShift);
+    Mshift.populate(phiShift);
```

Comment:
Were tests that depended on Mshift passing before?  Because this seems like a meaningful change and if they were passing before that's a mystery, isn't it?

Reply:
This test never worked the right way; nothing to explain.